apps/ideaspace/src/components/common/IdeaCard/IdeaCardTag.js

```diff
@@ -1,31 +1,43 @@
 import React, { useState } from 'react';
-import theme from '@devlaunchers/components/styles/theme';
+// import theme from '@devlaunchers/components/styles/theme';
 import { StatuBox } from './StyledIdeaCard';
 
 const IdeaCardTag = ({ status }) => {
   const [tag, setTag] = useState('');
-  const [tagColor, setTagColor] = useState('');
+  // const [tagColor, setTagColor] = useState('');
 
-  React.useEffect(() => {
-    if (status == 'submitted') {
-      setTag('submitted');
-      setTagColor(theme.colors.LIGHT_BLUE_100);
-    } else if (status == 'applying') {
-      setTag('in-review');
-      setTagColor(theme.colors.YELLOW_100);
-    } else if (status == 'approved') {
-      setTag('approved');
-      setTagColor(theme.colors.SUCCESS_100);
-    } else if (status == 'archived') {
-      setTag('archived');
-      setTagColor(theme.colors.BLUE);
-    } else {
-      setTag('workshopping');
-      setTagColor(theme.colors.ORANGE_100);
-    }
-  }, [status]);
+  const tagColor = {
+    submitted: 'bg-light-blue-lightblue-100',
+    'in-review': 'bg-yellow-yellow-100',
+    approved: 'bg-success-success-100',
+    archieved: 'bg-blue-blue-100',
+    workshopping: 'bg-orange-orange-100',
+  };
```

Comment:
these should be typed like this:
```js
const tagColor = {
    submitted: 'bg-LightBlue-lightblue-100',
    'in-review': 'bg-Yellow-yellow-100',
    approved: 'bg-Success-success-100',
    archieved: 'bg-Blue-blue-100',
    workshopping: 'bg-Orange-orange-100',
  };
```
I fixed it in the last commit so you just need to pull